rotkehlchen/chain/ethereum/modules/airdrops/decoder.py

```diff
@@ -251,6 +275,7 @@ def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
             CONVEX: (self._decode_fpis_claim, 'convex'),
             FOX_DISTRIBUTOR: (self._decode_fox_claim,),
             ELFI_LOCKING: (self._decode_elfi_claim,),
+            COW_PROTOCOL: (lambda context: decode_cow_claim(context, self.vcow),),
```

Comment:
can you check if in gnosis and here it works to do

```suggestion
            ELFI_LOCKING: (self._decode_elfi_claim,),
            COW_PROTOCOL: (decode_cow_claim,  self.vcow),
```

if I recall correctly the first element is the decoding logic and then you have the args for the function excluding the context